pyproject.toml

```diff
@@ -28,6 +28,7 @@ python-dateutil = '*'
 pandas = '*'
 pyproj = '>=2.6'
 requests = '*'
+searvey = { git = 'https://github.com/oceanmodeling/searvey.git' }
```

Comment:
here is the dependency on `searvey`; it should be changed to PyPI when `searvey` is released

Reply:
Just keep in mind that you can't publish stormevents to PyPI if it has git dependencies. PyPI does not allow it. 

Nevertheless, we should be really close to releasing searvey on PyPI, though. Probably later this week.

Reply:
Oh I did not know that, thanks